validator/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "amp_validator",
+  "version": "0.1.0",
+  "description": "Validator for AMP HTML (www.ampproject.org)",
+  "main": "index.js",
```

Comment:
There is one, but not included in this PR. The purpose of package.json is to let build.py fetch the dependencies, so index.js isn't used for anything, and build.py contains a snippet which generates its own main method, available (when built) as ./codegen/validate. I'll remove this line and see how it goes.